Src/Web/Web.Shared.Net/RequestTrackingTelemetryModule.cs

```diff
@@ -22,6 +23,31 @@ public class RequestTrackingTelemetryModule : ITelemetryModule
         private bool correlationHeadersEnabled = true;
         private string telemetryChannelEnpoint;
         private CorrelationIdLookupHelper correlationIdLookupHelper;
+        private ChildRequestTrackingSuppressionModule childRequestTrackingSuppressionModule = null;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="RequestTrackingTelemetryModule" /> class.
+        /// </summary>
+        public RequestTrackingTelemetryModule() : this(enableChildRequestsSuppression: true)
```

Comment:
This looks like a change in Public API, we would need to mention it in Changelog and in the Public API reference file (we have one in dotnet repo, not sure if there is one in dotnet-server repo, though...).
We might want to consider other implementations if this exposure is not necessary and/or make it configurable.

Reply:
The public constructor is unchanged, the variable is passed to an internal constructor.
I don't think any documentation would be required.

I like the idea of making it configurable, let's discuss.

Reply:
You can make it configurable by exposing public property. It is a good idea so if something goes south on specific environment we can shut down this logic completely